frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -431,9 +433,11 @@ export const PersistentVolumeClaimsPage = (props) => {
     />
   );
 };
+
 export const PersistentVolumeClaimsDetailsPage = (props) => (
   <DetailsPage
     {...props}
+    getResourceStatus={PVCStatus}
```

Comment:
We shouldn't be passing a react component as a value for this prop. `getResourceStatus` should return a string.

Reply:
> We shouldn't be passing a react component as a value for this prop. `getResourceStatus` should return a string.

Will address. Thanks.